Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/PDFDataDetectorOverlayController.mm

```diff
@@ -247,7 +247,7 @@
 
 void PDFDataDetectorOverlayController::updatePlatformHighlightData(PDFDocumentLayout::PageIndex pageIndex)
 {
-    WTF::forEach(m_pdfDataDetectorItemsWithHighlightsMap.get(pageIndex), [&](auto& dataDetectorItemWithHighlight) {
+    std::ranges::for_each(m_pdfDataDetectorItemsWithHighlightsMap.get(pageIndex), [&](auto& dataDetectorItemWithHighlight) {
```

Comment:
I think range-for would be better here as well. It would probably even be a bit better since you can inline the destructuring

```c++
for (auto [dataDetectorItem, coreHighlight] : m_pdfDataDetectorItemsWithHighlightsMap.get(pageIndex))
    coreHighlight->setHighlight(createPlatformDataDetectorHighlight(Ref { dataDetectorItem }).get());
```

(or something like that).